pkg/api/unversioned/deep_copy_generated.go

```diff
@@ -21,127 +21,15 @@ limitations under the License.
 package unversioned
 
 import (
-	"time"
-
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	time "time"
 )
 
-func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c *conversion.Cloner) error {
```

Comment:
I think we should be defining which packages should have deep copy
generated in a more structured way.  We have to *stop* traversal at the
boundaries of the Kube root because we can't change code upstream.  The way
proto does this is flood fill the dependency tree and create deep copies as
necessary at each edge (whenever a package exits the set of listed
packages).  That's a bit harder to implement, since you have to calculate
what packages to generate.

On Mon, May 23, 2016 at 9:50 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In pkg/api/unversioned/deep_copy_generated.go
> <https://github.com/kubernetes/kubernetes/pull/26051#discussion_r64225429>
> :
>
> >  )
> >
> > -func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c *conversion.Cloner) error {
>
> hmm - actually I've just started wondering why there were exposed before
> my change and now they are no longer exposed. And to be honest, now I don't
> understand why.
> [Those types comes from k8s.io/kubernetes and all their fields also...]
> Any hints?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26051/files/ee463c073a7f6853fc2a6019de435aa5d4e6c2c4#r64225429>
>